mesonbuild/interpreterbase.py

```diff
@@ -260,14 +260,14 @@ def evaluate_andstatement(self, cur):
     def evaluate_orstatement(self, cur):
         l = self.evaluate_statement(cur.left)
         if isinstance(l, mparser.BooleanNode):
-            l = l.get_value()
+            l = l.value
```

Comment:
Actually, I think this might be dead code, since `evaluate_statement` should have turned anything from a `Node` into a concrete value.